styles/wishlist.css

```diff
@@ -0,0 +1,92 @@
+@import url("../styles/main.css");
+
+.head-title {
+  text-align: center;
+  margin-top: 1rem;
+}
+.wishlist-items {
+  display: flex;
+  justify-content: center;
+}
+.basic-card {
+  display: flex;
+  flex-direction: column;
+  width: 18rem;
+  margin: 1rem;
+  font-family: "Roboto", sans-serif;
+  box-shadow: 20px 20px 50px grey;
+  border-radius: 5px;
+  position: relative;
+}
+.card-title {
+  padding: 10px;
+}
+
+.title {
+  color: #323232;
+}
+
+.author {
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.content {
+  padding-left: 10px;
+  font-size: 12px;
+}
+
+.price-container {
+  padding-left: 10px;
+  margin-top: 10px;
+}
+.price {
+  color: red;
+  margin-top: 2rem;
+  font-size: 1.3rem;
+}
+footer {
+  display: flex;
+  justify-content: space-between;
```

Comment:
One suggestion is don't use html tag names for styles

Reply:
ok